### ISSUE TITLE
transformers-loadable export (trust_remote_code)

### ISSUE THREAD

**Issue by MOBIRIZER-TECHNOLOGY** [MOBIRIZER-TECHNOLOGY/AnuLM#13]
Add modeling_anulm.py and configuration_anulm.py so the Hub repos load with AutoModelForCausalLM(trust_remote_code=True). TODO.md item 10.

**Comment by MOBIRIZER-TECHNOLOGY** [comment]
Done in c4b7cee.

```python
from transformers import AutoModelForCausalLM, AutoTokenizer
m = AutoModelForCausalLM.from_pretrained("toonist/AnuLM-Coder-400M", trust_remote_code=True)
t = AutoTokenizer.from_pretrained("toonist/AnuLM-Coder-400M")
print(t.decode(m.generate(**t("def is_prime(n):", return_tensors="pt"), max_new_tokens=60)[0]))
```

`modeling_anulm.py` builds the real modules from `model.py` and adopts them under their own names, so parameter names match `model.safetensors` exactly — all four exports load with zero missing and zero unexpected keys. It adds only the HF surface: config, logits at every position, `CausalLMOutputWithPast`, and the native per-layer cache through `generate` (a `DynamicCache` cannot hold it, since MLA caches a latent plus one shared rope key).

Two traps that would otherwise have shipped silently, both now covered by tests:

- **Rotary tables are not in the checkpoint.** They are computed, so `model.py` registers them non-persistently; transformers built the model on the meta device and materialised them as uninitialised memory. Every weight loaded, nothing warned, logits were wrong by 5.4.
- **bfloat16 weights break the model.** The router's `expert_bias` is kept in float32 on purpose; rounding it to 16 bits changes which experts fire, and `def is_prime(n):` came back as `<|endoftext|> set() set() set()...`. The configs declare float32 now and the wrapper warns on an override.

Verified against the native path: same logits to 1e-4, identical greedy output cached and uncached. `tools_hf_upgrade.py` applied it to the four repos already on the Hub — six small files each, weights untouched.
